examples/lighting-app/esp32/sdkconfig.defaults.esp32c61

```diff
@@ -0,0 +1,83 @@
+CONFIG_IDF_TARGET="esp32c61"
```

Comment:
please add support for build_examples.py as well. In https://github.com/project-chip/connectedhomeip/blob/master/scripts/build/builders/esp32.py#L24 and https://github.com/project-chip/connectedhomeip/blob/master/scripts/build/build/targets.py#L208

Reply:
Done.